Framework/Core/Windows/Devices/Pwm/PwmController.cs

```diff
@@ -0,0 +1,86 @@
+using System;
+using Windows.Devices.Pwm.Provider;
+
+namespace Windows.Devices.Pwm
+{
+    public sealed class PwmController
+    {
+        private IPwmControllerProvider m_provider;
+        private double m_minFrequency;
+        private double m_maxFrequency;

```

Comment:
why you defined m_manFrequency and m_maxFrequency? similar to the other properties, you could get them from m_provider whenever you want them.